doc/whats_new.rst

```diff
@@ -23,6 +23,7 @@ Changelog
 ~~~~~~~~~
 
 - :func:`read_raw_bids` now reads in participants tsv data, by `Adam Li`_ (`#392 <https://github.com/mne-tools/mne-bids/pull/392>`_)
+- :func:`mne_bids.utils.get_entity_vals` has gained ``ignore_*`` keyword arguments to exclude specific datasets from the list of results, by `Richard Höchenberger`_ (`#409 <https://github.com/mne-tools/mne-bids/pull/409>`_)
```

Comment:
```suggestion
- :func:`mne_bids.utils.get_entity_vals` has gained ``ignore_*`` keyword arguments to exclude specific entities from the list of results, by `Richard Höchenberger`_ (`#409 <https://github.com/mne-tools/mne-bids/pull/409>`_)
```

Reply:
Haha this is _exactly_ the phrasing I had used in my first iteration and then changed it to what we have now… felt that it's confusing / misleading:
because the function is called `get_entity_vals` and takes as an argument the _entity whose values you want to retrieve_. So when the docstring now says, "...exclude specific entities", it's kind of unclear wth is meant by that, right? because the only _entity_ you've dealt with until then is the one you specified in the function call… you know what I mean?

Reply:
But I do agree that "datasets" is also not the best word here!

Reply:
I see, I think you could even say:

"to exclude specific values from the list of results, for example the 'emptyroom' from the list of subjects".

This is user-facing, so the more clear we are, the better.